examples/object_detection/dataflow/coco128.py

```diff
@@ -102,9 +106,14 @@ class COCO128Dataset(Dataset):
     classes = list(label_to_name.values())
     name_to_label = {v: k for k, v in enumerate(classes)}
 
-    def __init__(self, data_path, mode, buffer_size=None, return_yolo_dict=False):
+    def __init__(self, data_path, mode, buffer_size=64, download=False, return_yolo_dict=False):
```

Comment:
Do you remember what does `buffer_size=64` and why we need to set it here instead of None?

Reply:
it gets passed through and sets batch_size on a super.__init__ call... and fails if it is set to None.